bar_and_lounge/data/pos_config.xml

```diff
@@ -2,35 +2,30 @@
 <odoo noupdate="1">
     <record id="pos_config_2" model="pos.config">
         <field name="name">Porter (International Airport)</field>
-        <field name="module_pos_hr" eval="True" />
```

Comment:
Good to have added the dependency, but you also need to keep it in pos config (pos config does not work the same way as res.config.settings)